src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/controller/PrisonerProfileController.kt

```diff
@@ -10,16 +10,19 @@ import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.config.ErrorResponse
 import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.PrisonerProfileDto
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.prisoner.search.PrisonerBasicInfoDto
 import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.service.PrisonerProfileService
 
 const val ORCHESTRATION_PRISONER_CONTROLLER_PATH: String = "/prisoner"
+const val ORCHESTRATION_PRISONER_PROFILE_CONTROLLER_PATH: String = "$ORCHESTRATION_PRISONER_CONTROLLER_PATH/{prisonId}/{prisonerId}/profile"
+const val ORCHESTRATION_PRISONERS_BASIC_INFO_CONTROLLER_PATH: String = "$ORCHESTRATION_PRISONER_CONTROLLER_PATH/{prisonerIds}/basic-details"
 
```

Comment:
not sure about the nameing,... notmally w would have prisoners/3233223/prison/CFI something like that,..

Reply:
https://dsdmoj.atlassian.net/browse/VB-3734 raised to correct existing prisoner profile endpoint path